learn/what_is_meilisearch/README.md

```diff
@@ -47,8 +47,8 @@ Why should you use MeiliSearch instead of any other existing solution? We try to
 
 ## Give it a try!
 
-Instead of showing you examples, why not just invite you to test MeiliSearch interactively in the **out-of-the-box web interface** we deliver?
+Instead of showing you examples, why not just invite you to test MeiliSearch interactively in the **out-of-the-box search preview interface** we deliver?
 
-There's no need to write a single line of front-end code. All you need to do is follow [this guide](/reference/features/web_interface.md) to give the search engine a try!
+There's no need to write a single line of front-end code. All you need to do is follow [this guide](/learn/what_is_meilisearch/search_preview.md) to give the search engine a try!
 
 ![web interface](/web-interface.png)
```

Comment:
Should we rename this and the image as well?

Reply:
hmmm…I guess? It's not super necessary since very few users will see it, but I don't think we risk anything by updating it. Thanks!